Source/WebKit/UIProcess/WebPageProxy.h

```diff
@@ -28,6 +28,7 @@
 #include "APIObject.h"
 #include "MessageReceiver.h"
 #include <WebCore/FrameIdentifier.h>
+#include <WebCore/FrameLoaderClient.h>
```

Comment:
Please don't include more headers here.  WebPageProxy.h is included by so many things it will greatly increase our build time.  Instead, just forward declare it and make more loadRequest functions.  I'll send you a diff.